astroquery/mast/missions.py

```diff
@@ -133,6 +152,9 @@ def query_criteria_async(self, **criteria):
         coordinates = criteria.pop('coordinates', None)
         objectname = criteria.pop('objectname', None)
         radius = criteria.pop('radius', 0.2*u.deg)
+        self.limit = criteria.pop('limit', 5000)
+        offset = criteria.pop('offset', 0)
+        select_cols = criteria.pop('select_cols', [])
```

Comment:
Is there any particular reason that these 6 cannot be pulled out from criteria in the signature? I would think it would be beneficial for the reason to make the defaults visible for the users, and then the 6 lines could be dropped down to 2 in the code itself.